README.md

```diff
@@ -678,9 +678,11 @@ The encrypted binary will have the following structure:
 - Padding to ensure the encrypted length is a multiple of 4 words
 - Signature metadata block
 
-The AES key must be provided as a 1024 bit (128 byte) .bin file containing 4-way shares of each word of the 256 bit AES key to be used for encryption.
-This should usually be generated by creating a random file of length 128 bytes and deriving the AES key from that, rather than crafting a file to match a specific AES key, to ensure the shares are random.
+The AES key can either be provided as a 1024 bit (128 byte) .bin file containing 4-way shares of each word of the 256 bit AES key to be used for encryption, or as a 256 bit (32 byte) .bin file containing the 256 bit AES key.
+The key share should usually be generated by creating a random file of length 128 bytes and deriving the AES key from that, rather than crafting a file to match a specific AES key, to ensure the shares are random.
+When passing a 256 bit AES key (as opposed to a 1028 bit 4-way AES key share), picotool will generate a random key share to match that key, by generating 3 random shares (A, B & C) and calculating a 4th share (D) to match the provided key.
```

Comment:
IMHO this needs rewording for clarity, as it's currently pretty confusing with the two different methods intermingled. (Also, you've written "1028 bit" which I'm assuming is wrong :wink: )